pyne/ace.pyx

```diff
@@ -1555,10 +1735,121 @@ class Reaction(object):
         self.TY = None     # Neutron release
         self.IE = 0        # Energy grid index
         self.sigma = []    # Cross section values
-
-    def broaden(self, T_high):
-        pass
-
+        self.sqrt_pi_inv = 1. / np.sqrt(np.pi)
+
+    def broaden(self, t_high):
```

Comment:
Not sure how much you care about performance, but this method will run *much* faster with a few static type declarations via Cython